pandas/tests/extension/decimal/array.py

```diff
@@ -15,6 +15,17 @@ class DecimalDtype(ExtensionDtype):
     name = 'decimal'
     na_value = decimal.Decimal('NaN')
 
+    def __init__(self, context=None):
```

Comment:
Seemed like a good idea to have a parameterized ExtensionDtype in the tests folder. For `.astype`, we're testing that `Series[decimal].astype(DecimalDtype(new_context))` is passed all the way through.